### ISSUE TITLE
[Improvement] Substitute for missing icons

### ISSUE THREAD

**Issue by olaf4** [segler-alex/RadioDroid#244]
If the station does not include an icon, include the icon of the **domain** instead.

For radio stations without any icon, include a neutral **standard icon**.

**Comment by Cormak** [comment]
@olaf4 I agree with you on having a default icon for stations without an icon, but if you go to http://www.radio-browser.info you can add any icon you want to any station, and they are quite easy to find by looking at all the media on the station's homepage.

**Comment by PersianAstroman** [comment]
In my version of RadioDroid (0.69), when a station has no logo/icon, the logo/icon of the last station stays on the notification panel which I suppose is a bug.
